muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/workspace-materials-reading.js

```diff
@@ -18,7 +18,7 @@
           window.location.hash = 'p-' + workspaceMaterialId;
           scrollTop = $('#page-' + workspaceMaterialId).offset().top - topOffset;
           $('html,body').scrollTop(scrollTop);
-          $.waypoints('refresh');
+          Waypoints.refreshAll();
```

Comment:
The other calls were Waypoint.refreshAll() and this is Waypoints.refreshAll() so which is the correct one?